geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/UpdatePropagationDUnitTest.java

```diff
@@ -91,30 +96,70 @@ public final void postSetUp() throws Exception {
     // Server2 VM
     server2 = host.getVM(1);
 
+    // Server3 VM
+    server3 = host.getVM(2);
+
     // Client 1 VM
-    client1 = host.getVM(2);
+    client1 = host.getVM(3);
 
     // client 2 VM
-    client2 = host.getVM(3);
+    client2 = host.getVM(4);
 
     PORT1 = server1.invoke(() -> createServerCache());
     PORT2 = server2.invoke(() -> createServerCache());
+    PORT3 = server3.invoke(() -> createServerCache());
 
-    client1.invoke(
-        () -> createClientCache(NetworkUtils.getServerHostName(server1.getHost()), PORT1, PORT2));
-    client2.invoke(
-        () -> createClientCache(NetworkUtils.getServerHostName(server1.getHost()), PORT1, PORT2));
 
     IgnoredException.addIgnoredException("java.net.SocketException");
     IgnoredException.addIgnoredException("Unexpected IOException");
   }
 
+
+
+  @Test
+  public void updatesArePropagatedToAllMembersWhenOneKilled() {
+    client1.invoke(
+        () -> createClientCache(NetworkUtils.getServerHostName(server1.getHost()), PORT1));
+    client2.invoke(
+        () -> createClientCache(NetworkUtils.getServerHostName(server3.getHost()), PORT3));
+    int entries = 20;
+    AsyncInvocation invocation = client1.invokeAsync(() -> doPuts(entries));
```

Comment:
You should add a type and name the `AsyncInvocation` variable to be what it's doing. So something like:
```
AsyncInvocation<Void> doPutsAsync = client1.invokeAsync(() -> doPuts(entries));
```
Or if you're using a lot of `AsyncInvocations`, it can be helpful to include the VM or a name on the end like `doPutsAsyncInVM1` or `doPutsAsyncInClient1`.